backend/app/models/Role.py

```diff
@@ -0,0 +1,9 @@
+from sqlalchemy import Column, Integer, String
+
+from .Base import Base
+
+
```

Comment:
Can we add just a few comments above table definitions, giving a brief description of what the table is used for. Ie role tables will be like entitlements table housing all possible user roles on the system. Documentation on variables if we think that's important could also be added but might be overkill.